Assets/HoloToolkit/Input/Scripts/InputSources/CustomInputSource.cs

```diff
@@ -456,6 +480,11 @@ private void SendControllerStateEvents(float time)
                 else
                 {
                     InputManager.Instance.RaiseManipulationUpdated(this, controllerId, currentButtonStates.CumulativeDelta);
+
+                    //Navigation Gesture Emulation
+                    NavigatorValues.x = Mathf.Clamp(currentButtonStates.CumulativeDelta.x*5, -1.0f, 1.0f) * railUsedCurrently.x;
```

Comment:
what is the multiply by 5 accomplishing? is this something that should be configurable?

Reply:
changes to this file are from previous commit(s) by other developers.
Its not part of the Tooltip commit.

Reply:
You may wish to rebase against the may18_dev branch to eliminate files that you are not modifying. the branch was updated to "final" 2017.2.1.4 changes yesterday.